.circleci/config.yml

```diff
@@ -228,21 +237,27 @@ common-steps:
     run:
       name: Commit workstation debs for deployment to apt-test.freedom.press
       command: |
-        PLATFORM="$(lsb_release -sc)"
+        apt-get update
+        apt-get install -y ca-certificates git git-lfs openssh-client
+
         git clone git@github.com:freedomofpress/securedrop-dev-packages-lfs.git
         cd securedrop-dev-packages-lfs
 
         git config user.email "securedrop@freedom.press"
         git config user.name "sdcibot"
 
-        # Copy built debian packages to the relevant workstation repo and git push.
-        mkdir -p ./workstation/${PLATFORM}-nightlies/
-        cp ~/project/build/debbuild/packaging/*.deb ./workstation/${PLATFORM}-nightlies/
-        git add workstation/${PLATFORM}-nightlies/*.deb
-        git commit -m "Automated SecureDrop workstation build"
+        for codename in buster bullseye
```

Comment:
Future cleanup opportunity would be to try to specify this in variables so it's not hardcoded here.